obal/data/module_utils/koji_wrapper.py

```diff
@@ -0,0 +1,19 @@
+"""
+A koji wrapper
+"""
+from subprocess import STDOUT, check_output, CalledProcessError
+
+class KojiCommandError(Exception):
+    """Raised when Koji command fails"""
+
+def koji(command, executable):
+    """
+    Run a koji command
+    """
+    if not executable:
+        executable = 'koji'
```

Comment:
```suggestion
def koji(command, executable=None):
    """
    Run a koji command
    """
    if executable is None:
        executable = 'koji'
```

Python does not allow Ruby syntax ;)